### ISSUE TITLE
Some limits are not appearing

### ISSUE THREAD

**Issue by ddemaret-plaid** [aws-solutions/quota-monitor-for-aws#201]
**Describe the bug**

I've deploy the solution into multiple accounts but can't find any trace of any Amazon SageMaker or Amazon CloudFront quota's in the DynamoDB. More could be missing, I haven't checked them all. 

**To Reproduce**

Deploy the solution, then query the data that has been collected and look for any traces of Amazon SageMaker or Cloudfront quotas. For example "Cookies per cache policy".

**Expected behavior**

All quota's are collected in the dynamodb table QMTable.

**Please complete the following information about the solution:**

- [x] Version: [e.g. v1.0.0]
 v6.2.9
- [x] Region: [e.g. us-east-1]
us-east-1
- [x] Was the solution modified from the version published on this repository?
no

- [x] Have you checked your [service quotas](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html) for the services this solution uses?
- [x] Were there any errors in the CloudWatch Logs? [How to enable debug mode?](https://docs.aws.amazon.com/solutions/latest/quota-monitor-for-aws/troubleshooting.html)
None that I could find

**Screenshots**
If applicable, add screenshots to help explain your problem (please **DO NOT include sensitive information**).
Results from
```
SELECT distinct(service)
FROM "default"."quota-monitor-hub-qmtable336670b0-1n3u9ljhx0m6l"
```
```
"service"
"DynamoDB"
"Bedrock"
"Connect"
"SSM"
"Route53"
"DiagnosticTools"
"Kinesis"
"SNS"
"Location"
"ECR"
"SES"
"AppSync"
"Fargate"
"IoT"
"CloudWatch"
"NeptuneGraph"
"Lightsail"
"KMS"
"RDS"
"RoboMaker"
"VPC"
"CloudTrail"
"EMR"
"EBS"
"Braket"
"Textract"
"ELB"
"NetworkMonitor"
"Firehose"
"EventBridge"
"IVS"
"AutoScaling"
"FIS"
"IAM"
"Athena"
"CloudFormation"
"IVSChat"
"AppIntegrations"
"RolesAnywhere"
"TrustedAdvisor"
"Logs"
"EC2"
"CloudHSM"
```
**Additional context**
All services have been enabled in `SQServiceTable` including sagemaker and cloudfront.

**Comment by sanjay-reddy-kandi** [comment]
Thank you for reporting this issue and providing detailed information. We understand that you're unable to find any traces of Amazon SageMaker or Amazon CloudFront quotas in the DynamoDB table, despite having enabled these services in the SQServiceTable.

We will investigate this issue and work on identifying the root cause. In the meantime, we would appreciate if you could provide us with any additional information or logs(specifically QM-ListManager-Function logs) that might be helpful in troubleshooting this issue.

**Comment by ddemaret-plaid** [comment]
Sure. [Here's some logs](https://gist.github.com/ddemaret-plaid/da8b3c71e4a0eae62f34d9a84d7ebda6)

**Comment by sanjay-reddy-kandi** [comment]
When I run `aws service-quotas list-service-quotas --service-code cloudfront` , I see no quotas, so we do not monitor Cloudfront service. Even for the services supported by service quotas, we filter out metrics that do not have CloudWatch AWS/Usage. For the SageMaker and the Warnings in the logs, we are releasing a patch this week. Hope it helps!

**Comment by sanjay-reddy-kandi** [comment]
yeah this could be a bug or a temporary issue with the Service Quotas API for CloudFront.

**Comment by ddemaret-plaid** [comment]
Created an issue here: https://github.com/aws/aws-cli/issues/8941

**Comment by sanjay-reddy-kandi** [comment]
Just pointing it out again, Even if that cli works, we will not monitor cloudfront because its quotas have got no Utilization metrics

**Comment by ddemaret-plaid** [comment]
Aaah sorry I misunderstood. Is this because all utilisation metrics come from cloudwatch? And cloudfront metrics aren't there? (failing [this](https://github.com/aws-solutions/quota-monitor-for-aws/blob/cf286797b0f2981ec7d626ee2b6c3a15734c997c/source/lambda/utilsLayer/lib/servicequotas.ts#L186) check i assume) Would you have a list of which other metrics are missing and would you have any idea if they will ever be added?

**Comment by sanjay-reddy-kandi** [comment]
Yes that is the right code check. I don't know the specifics but Service quota team is the right team to answer that question but based our team discussion with the them, they are onboarding lot of AWS services

**Comment by sanjay-reddy-kandi** [comment]
This is issue is fixed in the latest version `v6.2.11`
